purebred.cabal

```diff
@@ -91,3 +91,5 @@ test-suite unittests
                      , resourcet
                      , time
                      , regex-posix
+                     , mtl
+                     , lens
```

Comment:
Not sure if that is a bit over the top really just to pluck out the third element of a tuple. Maybe should use a data type?

Reply:
nah go for it mate.  we're already depending on these in the lib/main program so it's no biggie.